pkg/retry/util.go

```diff
@@ -170,3 +172,11 @@ func recordRetryInfo(firstRI, lastRI rpcinfo.RPCInfo, callTimes int32, lastCosts
 		}
 	}
 }
+
+// IsRetryRequest checks whether it's a retry request by checking the RetryTag set in rpcinfo
+// It's supposed to be used in client middlewares
+func IsRetryRequest(ctx context.Context) bool {
```

Comment:
rename to IsLocalRetryRequest ? because `metainfo.GetPersistentValue(ctx,retry.TransitKey)` can check if it is a upstream retry request, this name cannot distinguish

Reply:
Updated. Also added a "IsRemoteRetryRequest".